messageapp/routes/messages.routes.js

```diff
@@ -16,18 +21,36 @@ router.post('/messages', (req, res, next) => {
 
     } else if (typeof destination != 'string' || typeof body != 'string') {
 
-        res.status(400).json({ message: "Fields must be filled with text" })
+        res.status(406).json({ message: "Fields must be filled with text" })
 
     } else {
 
         messagesService
             .sendMessage({ destination, body })
-            .then((message) => res.status(200).json(message.data))
-            .catch(err => res.status(500).json(err))
+            .then(() => {
+
+                messagesMongodb
+                    .saveMessage({ destination, body, number })
+                    .then(response => res.status(200).json({ message: "Message successfully recorded" }))
```

Comment:
Shouldn't update the state of the message? after it was 200?
and after there was an error?

Reply:
/resolved as the code was updated and the state was added.